### ISSUE TITLE
[Broken Website] redhat.com/en

### ISSUE THREAD

**Issue by q1800** [darkreader/darkreader#8541]
# Broken Website Report

## Website Address
https://www.redhat.com/en

## Steps To Reproduce
Visit the website with Dark Reader enabled.
Notice the logo in the screenshot needs fixing.
Notice the button outline in the screenshot needs fixing.

## Problematic Observation
Logo
Button Outline

## Screenshots
![Screenshot 2022-03-28 at 17-58-35 The world's open source leader](https://user-images.githubusercontent.com/95879668/160501916-92a4486f-81e9-4cce-b315-cfb2e94d57a3.png)

## System Information

- OS: Windows 10
- Browser: Firefox Nightly 100
- Dark Reader Version: 4.9.47

**Comment by Gusted** [comment]
What should be changed to the button outline?

**Comment by q1800** [comment]
Just to make it stand out more? I wasn't sure about having this fixed.

If you think it doesn't need fixed or is too nit-picky, I can understand.

I guess I just have a more general question about little nit-picky elements like this. Are they worth fixing or not really in the scope of Dark Reader? I honestly don't know either way. Please let me know your thoughts and what are the general best practices for Dark Reader on items like these,

**Comment by Gusted** [comment]
> Just to make it stand out more? I wasn't sure about having this fixed.
> 
> If you think it doesn't need fixed or is too nit-picky, I can understand.

Yeah... border colors are technically correct but the contrast isn't that great on this border.



> I guess I just have a more general question about little nit-picky elements like this. Are they worth fixing or not really in the scope of Dark Reader? I honestly don't know either way. Please let me know your thoughts and what are the general best practices for Dark Reader on items like these,

To clarify first(because it happens that people will make PR's based on something I said months/years ago and misinterpret it)
We have the _small enhancements_ fixes and the _cosmetic_ changes.

This is a clear and conceive case of a small enhancement, there's supposed to be a border color there(as the non-modified page has it) and in the current form, Dark Reader isn't able to have the correct contrast for this color. This would be considered a acceptable fix. 

A cosmetic fix would be when there was no border color on the non-modified page in the first place! Dark Reader users didn't install Dark Reader to let some user decide how certain elements should look, it should only care to modify the page into a custom dark color scheme.

Well I'm personally not a fan of small enhancements because they mostly boil down to some kind of opinionated color(The best option is to choose the direct "invert" of that color so nobody could argue with it or take it as a example to do the same for another site) and if someone really decides to give it a try the config can be cluttered with thousands of such small enhancements. But this is "big enough"(Boils down the opinion of the maintainer, aka me)

Like I said this is a simple and small case which is pretty acceptable to be fixed. But ^^^ That doesn't mean everything will get a fix, nothing personal, but I have to draw a line here so other people won't quote me on this and say "But you said this is okay! *Add some insult here*".

**Comment by q1800** [comment]
I completely understand. I appreciate and thank you for the clear no B.S. answer. I know it's nothing personal and you have to do what you have to do. 👍
